airflow/providers/amazon/aws/hooks/batch_client.py

```diff
@@ -39,10 +39,10 @@
 
 
 @runtime_checkable
-class AwsBatchProtocol(Protocol):
+class BatchProtocol(Protocol):
     """
     A structured Protocol for ``boto3.client('batch') -> botocore.client.Batch``.
-    This is used for type hints on :py:meth:`.AwsBatchClient.client`; it covers
+    This is used for type hints on :py:meth:`.BatchClient.client`; it covers
     only the subset of client methods required.
```

Comment:
I'm not 100% sure if this one should have been left along.  Please double check me here.